src/common/Button/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode, useMemo } from 'react';
+import './style.scss';
+
+interface Props {
+  href?: string
+  icon?: ReactNode
+  children: ReactNode
+}
+
+const Button = ({ children, href, icon }: Props) => {
+  const baseClassName = 'btn';
+
+  const content = useMemo(() => (
```

Comment:
Tohle useMemo mi nějak nedává smysl. Podle mě slouží jenom k tomu, aby se nepřerenderoval content když se změní href. To už je celkem jedno ne?

Reply:
Jo no. Zbytečně jsem optimalizoval pro případ, že by se content posílal do další komponenty, což se neděj. `useMemo` odebráno.

Reply:
Ok